src/pudl/transform/eiaaeo.py

```diff
@@ -193,9 +291,14 @@ def _decumulate(df: pd.DataFrame, decumulate_columns: list[str]) -> pd.DataFrame
         .pipe(_drop_aggregated_generation_data)
         .pipe(_collect_totals)
     )
+    # 1 BKWh = 1e12 Wh = 1e6 MWh
```

Comment:
This unit conversion cleanup was just... there for the taking. Not really related to the rest of this PR.